manifest.json

```diff
@@ -3,25 +3,30 @@
     "description": "__MSG_extension_description__",
     "version": "1.6",
     "manifest_version": 2,
```

Comment:
Did you change the manifest.json things that I mentionned? I still see them changed ...